extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamInfoItemExtractor.java

```diff
@@ -116,4 +116,14 @@
     @Nullable
     DateWrapper getUploadDate() throws ParsingException;
 
+
+    /**
+     * Get the video's short description.
+     *
+     * @return The video's short description or {@code null} if not provided by the service.
+     * @throws ParsingException if there is an error in the extraction
+     */
+    @Nullable
+    String getShortDescription() throws ParsingException;
```

Comment:
Why don't implement a default method here?
Would remove the need for implementing this in each service
```java
    @Nullable
    @Override
    public String getShortDescription() {
        return null;
    }
```